lib/reboot_cmds.py

```diff
@@ -82,13 +84,16 @@ def fix_pro_pkg_holds(cfg):
                         )
                     )
                     sys.exit(1)
-                cfg.notice_file.remove(
-                    "", messages.FIPS_SYSTEM_REBOOT_REQUIRED.msg
+                notices.remove(
```

Comment:
Those notices should be on the `/var/run/` folder. Should we still manually remove them ?